### PR TITLE
Added the Hero Name

### PR DESCRIPTION
**Pull request by Akshay-CodingArena**
Hero Name Akshay is added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# Meri-Daastan
\ No newline at end of file
+# Meri-Daastan
+
+# 1. My name is Akshay
```
Comment (Pro-coder-akshay) [inline README.md]: Add More Details like Age and Body Structure

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-# Meri-Daastan
\ No newline at end of file
+# Meri-Daastan
```
Comment (Pro-coder-akshay) [inline README.md:1]: Can also Suggest better title

**Review by Pro-coder-akshay — COMMENTED** [review]
Do these changes.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,3 @@
 # Meri-Daastan
 
-# 1. My name is Akshay
+# 1. My name is Akshay. My age is 21 and 5'8 tall
```
Comment (Pro-coder-akshay) [inline README.md]: This is fine

File: README.md (hunk 2 above)
Comment (Akshay-CodingArena) [reply README.md:1]: Dastaan is good

**Review by Pro-coder-akshay — COMMENTED** [review]
I will Merge now

**Review by Pro-coder-akshay — CHANGES_REQUESTED** [review]
Do let me know

File: README.md (hunk 3 above)
Comment (Pro-coder-akshay) [reply README.md]: Can Add More

**Review by Pro-coder-akshay — CHANGES_REQUESTED** [review]
ASAP

File: README.md [hunk 4]
```diff
@@ -1,3 +1,3 @@
 # Meri-Daastan
 
-# 1. My name is Akshay. My age is 21 and 5'8 tall
+# 1. My name is Akshay. My age is 21 and 5'8 tall. Fair Skin
```
Comment (Pro-coder-akshay) [inline README.md:3]: Looking good

**Review by Pro-coder-akshay — APPROVED** [review]
Ready to merge

**Comment by Pro-coder-akshay** [comment]
Added Hero Description

**Pull request merged**